checks.d/docker_daemon.py

```diff
@@ -249,6 +254,21 @@ def check(self, instance):
         if self.collect_container_size:
             self._report_container_size(containers_by_id)
 
+        if self.collect_container_count:
+            self._report_container_count_by_state(containers_by_id)
+
+        if self.collect_dead_container_count:
+            self._report_container_count_by_state(containers_by_id, state="Dead")
+
+        if self.collect_exited_container_count:
+            self._report_container_count_by_state(containers_by_id, state="Exited")
+
+        if self.collect_volume_count:
+            self._report_volume_count()
+
+        if self.collect_dangling_volume_count:
+            self._report_volume_count(filters={'dangling': True})
```

Comment:
Looks like, dangling volumes will already be collected when you call collect_volume_count line 266.

We should likely reuse that so we don't do 2 API calls to the docker API.